package.json

```
@@ -18,41 +18,44 @@
   },
   "private": true,
   "dependencies": {
-    "@nx/devkit": "17.2.8",
-    "@swc/helpers": "~0.5.2",
+    "@nx/devkit": "19.8.2",
```

I think we have to stay to v17.2.8, no?
```suggestion
    "@nx/devkit": "17.2.8",
```